### ISSUE TITLE
Issue with automatic redirect

### ISSUE THREAD

**Issue by lewislbr** [wiziple/gatsby-plugin-intl#9]
Hi, first of all thanks for this plugin, I like the fact that allows a single page pull data rather than one page for each language.

Overall it works very well, creating pages for each language, but I'm having issues with the automatic redirect, as always goes to the default language, although I can change it later.

I'm looking for an automatic, non changeable option where it only shows the language based on the browser language like:

```
<script>
      let language = window.navigator.language;
      if (language == "es") {
        window.location.href = "/es";
      }
      if (language == "en") {
        window.location.href = "/en";
      }
    </script>
```

As adding the previous snippet to the helmet causes a load loop, is there a way to achieve it?

Thanks!

Repo: https://github.com/lewislbr/lewis-llobera-photography-gatsby

**Comment by wiziple** [comment]
Thanks for your report.
I also found this issue and fixed it on v0.1.7.
Please update your `gatsby-plugin-intl`.
Let me know if there's any other issue. 👍

**Comment by lewislbr** [comment]
Thanks, works great now! ✨
